### PR TITLE
Add the cmake option to checkout modules.

### PR DESCRIPTION
**Pull request by Shylock-Hg**

### CODE REVIEW

File: cmake/FetchModule.cmake [hunk 1]
```diff
@@ -30,34 +30,36 @@ macro(nebula_fetch_module)
         endif()
     else()
         message(STATUS "Updating from ${module_URL}")
```
Comment (laura-ding) [inline cmake/FetchModule.cmake:32]: here can be delete

File: cmake/FetchModule.cmake [hunk 2]
```diff
@@ -9,7 +9,7 @@ macro(nebula_fetch_module)
     cmake_parse_arguments(
         module                      # <prefix>
```
Comment (laura-ding) [inline cmake/FetchModule.cmake:10]: alignment

File: cmake/FetchModule.cmake (hunk 1 above)
Comment (Shylock-Hg) [reply cmake/FetchModule.cmake:32]: Fixed.

File: cmake/FetchModule.cmake (hunk 2 above)
Comment (Shylock-Hg) [reply cmake/FetchModule.cmake]: Fixed.

**Review by laura-ding — APPROVED** [review]

**Review by nevermore3 — APPROVED** [review]

**Pull request merged**